README.md

```diff
@@ -364,6 +366,40 @@ If a `.spec.type` is set, the `type` entry in the binding `Secret` **MUST** be s
 
 If the modification of the Application resource is completed successfully, the `Ready` condition status **MUST** be set to `True`.  If the modification of the Application resource is not completed successfully the `Ready` condition status **MUST NOT** be set to `True`.
 
+## Reconciliation Duties
+
+There are scenarios where the reconciler that processes the bindings of a provisioned service is different than the reconciler that will project those bindings into the Application. In such cases the `ServiceBinding` CR author **MUST** set `.spec.application.autoProjection` to `false`, signalling the separation of reconciliation duties between `.spec.service` and `.spec.application`.  The default value of this property is `true`.  
```

Comment:
As we discussed on the issue, booleans in APIs are often problematic. With Go, defaulting the value to `true` will require the type be `*bool` which will make it harder to use and programmatically create.

Reply:
I guess I am looking purely from a spec (language agnostic) perspective, where a boolean is a simple primitive that requires less learning.  The fact that we came up with so many variations for the Enums, none unanimously accepted, hinted to me that perhaps this required a simpler solution.

Reply:
As you mentioned, naming is indeed hard, but it's worth taking a couple iterations to get right. What I like about an enum is that it's a natural point for extension in the future.

For example, today we support projecting into PodSpec-able resources. If down the road there is a new resource shape we want to support, or a PodTemplateSpec subresource hosted by the k8s API Server, then it would be nice to have a way to instruct to the operator which strategy to use for the projection.

Reply:
that's a fair point.  I will revisit the Enum idea.